mteb/tasks/Clustering/fr/MLSUMClusteringP2P.py

```diff
@@ -13,7 +13,7 @@ class MLSUMClusteringP2P(AbsTaskClustering):
         name="MLSUMClusteringP2P",
         description="Clustering of newspaper article contents and titles from MLSUM dataset. Clustering of 10 sets on the newpaper article topics.",
         reference="https://huggingface.co/datasets/mlsum",
-        hf_hub_name="mteb/mlsum",
+        hf_hub_name="mteb/mlsum",  # can't find the original dataset on HF
```

Comment:
@imenelydiaker

Reply:
It's just `mlsum`, here is the link: https://huggingface.co/datasets/mlsum
```suggestion
        hf_hub_name="mlsum",  
```

Reply:
I'll open another PR to add the other languages that are supported by this dataset (de, en, etc.)

Reply:
Perfect @imenelydiaker thanks for the fast response.